docs/source/generating_column_data.rst

```diff
@@ -55,18 +58,21 @@ specification.
 When building the data generation spec, the ``withSchema`` method may be used to add columns from an existing schema.
 This does _not_ prevent the use of ``withColumn`` to add new columns.
 
-Use ``withColumn`` to define a new column. This method takes a parameter to specify the data type.
-See :data:`~dbldatagen.data_generator.DataGenerator.withColumn`.
+| Use ``withColumn`` to define a new column. This method takes a parameter to specify the data type.
+| See :data:`~dbldatagen.data_generator.DataGenerator.withColumn` for further details.
```

Comment:
"See the method" here as well to stay consistent with the previous comment?

Reply:
fixed